teleduck/test_sql_init.py

```
@@ -0,0 +1,69 @@
+    run_server(db_file, port, sql_scripts=scripts, sql=sqls)
+
+
+class SqlInitTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.port = 55442
+        fd, cls.db_file = tempfile.mkstemp(suffix=".db")
+        os.close(fd)
+        os.unlink(cls.db_file)
+
+        cls.script1 = tempfile.NamedTemporaryFile(delete=False, suffix=".sql")
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)